docs/examples/control/example_custom_keys.py

```diff
@@ -0,0 +1,33 @@
+from compas.geometry import Point, Polyline, Bezier
+from compas_view2.app import App
+
+# If you input incomplete configuration, the rest will be filled by the default values.
+# The default controller configuration is more Rhino-user friendly.
+# Below we provide a more ***Blender-user friendly*** configuration.
+
+
+config = {
+    "controller": {
+        "actions": {
+            "mouse_key": {
+                "pan": {"mouse": "middle", "key": "shift"},
+                "rotate": {"mouse": "middle", "key": ""},
+                "box_selection": {"mouse": "left", "key": ""},
+                "box_deselection": {"mouse": "left", "key": "control"},
+                "selection": {"mouse": "left", "multi_selection": "shift", "deselect": "control"},
+            },
+            "keys": {
+                "zoom_selected": ["control","."]
+            },
+        },
+    }
+}
+
+viewer = App(config=config)
+
+curve = Bezier([[0, 0, 0], [3, 6, 0], [5, -3, 0], [10, 0, 0]])
+pointobj = viewer.add(Point(*curve.point(0)), pointsize=20, pointcolor=(1, 0, 0))
+curveobj = viewer.add(Polyline(curve.locus()), linewidth=2)
```

Comment:
can we change this part of example to something that is easy to click on? maybe bunch of boxes

Reply:
Done
![image](https://github.com/compas-dev/compas_view2/assets/57562250/be763b40-c4a7-440f-98cd-a35e6eb3eaae)